### PR TITLE
Add FAQ tooltip

### PR DESCRIPTION
**Pull request by shulijones**
Add a brief explanation of the site that appears when a user hovers over a ? button in the top right corner.

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -20,7 +21,22 @@
 <body onload="registerServiceWorker(); createStateDropdownList();">
   <main>
     <div id="header">
-      <h1>Census Viz</h1>
+      <h1>Census Viz
+        <div class="tooltip">
+          <i class="material-icons" id="faq">help_outline</i>
```
Comment (nsri-google) [inline src/main/webapp/index.html:26]: Maybe add a one line summary of what censusviz is

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -20,7 +21,22 @@
...
+          <p class="tooltiptext">To use Census Viz, start typing in any box. The
```
Comment (nsri-google) [inline src/main/webapp/index.html:27]: maybe something like "select one of the available options from each drop down list and click the Green Go button. You can also start typing in any drop down to filter the options"

Comment (shulijones) [reply src/main/webapp/index.html:27]: That's a good suggestion, thank you. Done

File: src/main/webapp/index.html (hunk 1 above)
Comment (shulijones) [reply src/main/webapp/index.html:26]: This is a good idea! I will discuss our "elevator pitch" with Kyra and Olivia and add in the best one-sentence summary we can come up with.

File: src/main/webapp/index.html [hunk 3]
```diff
@@ -20,7 +21,22 @@
...
+            dropdown list will show you the options that are available. You
+            can mix and match between the lists to get answers to many
+            different questions. 
+            <br><br>
+            The data you see comes from several sources within the census.
+            Depending on your question, we choose the census data with the most
+            information available. Most of our data come from the annual
```
Comment (kyracq) [inline src/main/webapp/index.html:34]: most of our data comes

Comment (shulijones) [reply src/main/webapp/index.html:34]: Fixed, thanks

File: src/main/webapp/index.html (hunk 1 above)
Comment (shulijones) [reply src/main/webapp/index.html:26]: Done

**Pull request merged**